examples/glue_benchmark/README.md

```diff
@@ -0,0 +1,98 @@
+# GLUE Finetuning Script
+
+This script is written to help you evaluate your model on GLUE benchmarking.
+It provides the functionalities below:
+
+- GLUE dataset loading & common preprocessing
+- Finetuning flow 
+- Generate GLUE submission files
+
+To use the script, you need to change the code to load your pretrained model,
+and run the command below:
+
+```shell
+python glue.py --task_name="mrpc" --batch_size=32 \
+    --submission_file_path="glue_submissions/"
+```
+
+By default the script finetunes on KerasNLP BERT model 
+`keras_nlp.models.Bert.from_preset("bert_tiny_uncased_en")`.
+
+To make a real GLUE leaderboard submission, you need to call the finetuning on 
+all tasks, then enter the submission directory then zip the submission files::
+```shell
+for task in cola sst2 mrpc rte mnli_matched mnli_mismatched ax qnli qqp; do
+  python glue.py --task_name="$task" --submission_directory="glue_submissions/"
+done
+cd glue_submissions
+zip -r submission.zip *.tsv
+```
+
+GLUE submission requires the `submission.zip` contains `.tsv` file for all tasks
+, otherwise it will be a failed submission. An empty `.tsv` will also fail 
+because it checks the content. If you only want to evaluate on certain tasks, 
+you can download the sample submission, and put the `.tsv` files for tasks you 
+don't run inside your submission file. For example if you don't want to 
+run the `ax` task, then you can do:
+
+```
+curl -O https://gluebenchmark.com/assets/CBOW.zip
+unzip CBOW.zip -d sample_submissions
+cp sample_submissions/AX.tsv glue_submissions
+```
+
+## How to Use the Script
+
+To use this script on your model, you need to do 3 things:
+
+1. Implement your custom preprocessing in `preprocess_fn()`.
+2. Load your pretrained model.
+3. Make the finetune model with your model.
+
+Code needing customization is wrapped between comment
+`Custom code block starts` and 
+`Custom code block ends`. See instructions on each step below.
+
+### Custom Preprocessing
+
+In all GLUE dataset, each record comes with features of one or two sentences, 
+and one label. In the script, we load GLUE dataset in the format 
+`(features, labels)`,  where `features` is a tuple of either 1 sentence or 2
+sentences. Your need to write custom preprocessing logic to convert to data
+to the required input of your model. For example, in the current script 
+(finetuning for KerasNLP BERT), it is doing:
+
+```python
+bert_preprocessor = keras_nlp.models.BertPreprocessor.from_preset(
```

Comment:
I'm not sure this code block is adding much (they will see this in the code). Maybe just keep the paragraph?

Reply:
The paragraph is a bit abstract, so I want to show the actual code here.